### PR TITLE
Cabinet random filling fix

### PR DESCRIPTION
**Pull request by lapatison**
## About the PR

Office paper spawn prob-ability has been replaced with amount field

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml [hunk 1]
```diff
@@ -109,8 +109,8 @@
       - id: Pen
         prob: 0.5
       - id: PaperOffice
-      - id: PaperOffice
-        prob: 0.5
+        amount: 1
+        amountMax: 2
       - id: PaperOffice
         prob: 0.4
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:115]: at this point get rid of this and just make `amountMax` 3 and re-adjust the folder probability

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml (hunk 1 above)
Comment (lapatison) [reply Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:115]: should the `prob` sum be exactly 1.0?

Comment (AJCM-git) [reply Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:115]: for the folders? yeah

Comment (lapatison) [reply Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:115]: Can I just remove `prob` and expect that probability will be equally divided within the group?

Comment (AJCM-git) [reply Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:115]: that would just make all folders always spawn since prob is by default 1

Comment (lapatison) [reply Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml:115]: Oh no, I just noticed that I wrote `amountMax ` instead `maxAmount`. I'll fix it asap

Comment (lapatison) [reply Resources/Prototypes/Entities/Structures/Storage/filing_cabinets.yml]: Screenshot_1
I have also found that removing probability works just fine with orGroup.

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**